deploy/manifests/dev/us-east-2/tenant/storetheindex/indexstar/canary/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+resources:
+  - ../../../../../base/indexstar
+  - ingress.yaml
+  - pod-monitor.yaml
+
+patchesStrategicMerge:
+  - deployment.yaml
+  - pdb.yaml
+  - service.yaml
+
+replicas:
+  - name: indexstar-canary
+    count: 1
+
+images:
+  - name: indexstar
```

Comment:
* name should match the name in `Deployment` object, which is set to `indexstar-canary`
* I recommend moving this to the parent `kustomization` so that all images are overridden in a single kustomization.
* If they should always run the same version then i would use the same image `name` for both and specify it once.

Reply:
Moved to parent